capsules/src/rf233.rs

```diff
@@ -176,14 +180,69 @@ pub struct RF233<'a, S: spi::SpiMasterDevice + 'a> {
     tx_len: Cell<u8>,
     tx_client: Cell<Option<&'static radio::TxClient>>,
     rx_client: Cell<Option<&'static radio::RxClient>>,
+    cfg_client: Cell<Option<&'static radio::ConfigClient>>,
     addr: Cell<u16>,
     pan: Cell<u16>,
+    tx_power: Cell<i8>,
+    channel: Cell<u8>,
     seq: Cell<u8>,
     spi_rx: TakeCell<'static, [u8]>,
     spi_tx: TakeCell<'static, [u8]>,
     spi_buf: TakeCell<'static, [u8]>,
 }
 
+fn setting_to_power(setting: u8) -> i8 {
```

Comment:
"setting" is the value in the register, as specified by the datasheet (Table 9-9).